src/mon/NVMeofGwMap.cc

```diff
@@ -873,6 +875,29 @@ struct CMonRequestProposal : public Context {
   }
 };
 
+void NVMeofGwMap::get_health_checks(health_check_map_t *checks) const 
```

Comment:
since you're looking at the membership of a gateway group, would it make sense to also add a healthcheck for when a gateway is down/inaccessible here - that's the scenario that will hurt the most, IMO